src/views/settings/settings.view.tsx

```diff
@@ -0,0 +1,68 @@
+import { useState } from "react";
+
+import useSettingsStyles from "src/views/settings/settings.styles";
+import Typography from "src/views/shared/typography/typography.view";
+import Header from "src/views/shared/header/header.view";
+import Card from "src/views/shared/card/card.view";
+import { ReactComponent as ConversionIcon } from "src/assets/icons/currency-conversion.svg";
+import { ReactComponent as CheckedIcon } from "src/assets/icons/checkbox-checked.svg";
+import { ReactComponent as UncheckedIcon } from "src/assets/icons/checkbox-unchecked.svg";
+import { ReactComponent as LogoutIcon } from "src/assets/icons/logout.svg";
+import { ReactComponent as EurIcon } from "src/assets/currencies/eur.svg";
+import { ReactComponent as UsdIcon } from "src/assets/currencies/usd.svg";
+import { ReactComponent as JpyIcon } from "src/assets/currencies/jpy.svg";
+import { ReactComponent as GbpIcon } from "src/assets/currencies/gbp.svg";
+import { ReactComponent as CnyIcon } from "src/assets/currencies/cny.svg";
+
+const Settings = () => {
+  const classes = useSettingsStyles();
+  const [currencySelected, setCurrencySelected] = useState("eur");
```

Comment:
`const [selectedCurrencyId, setSelectedCurrencyId] = useState("eur");`

Reply:
BTW, I think that the type of `selectedCurrencyId` is `string`. We should create an enum or a type for all the currency ids: eur, usd, gbp, cny and jpy and make `selectedCurrencyId` use this type instead of an string.